src/app/main/_component/ChipContainer.tsx

```diff
@@ -11,9 +13,17 @@ const channelData = [
   { name: '공부', path: 'study' },
 ];
 
-const currentPath = 'all';
+interface ChipContainerProps {
+  currentChannel: string;
+}
+
+const ChipContainer = ({ currentChannel }: ChipContainerProps) => {
+  const [currentPath, setCurrentPath] = useState(currentChannel || 'all');
```

Comment:
모순이 있는 것 같아 리뷰를 남겨놓습니다.

currentChannel이 없다면 'all'이 state의 초기값이 되는것일텐데 props가 선택속성이 아니기에 currentChannel 이 빈값이 아닐것으로 예상됩니다.

이런경우라면 currenChannel은 선택송성으로 변경하고 currenChannel의 초기값에 'all'을 부여해주면 어떨까요 ?? 
```suggestion
const ChipContainer = ({ currentChannel  = 'all'}: ChipContainerProps) => {
  const [currentPath, setCurrentPath] = useState(currentChannel);
```
과 같이 가면 좋겠네요

Reply:
좋습니다!👍